figure/settings.py

```diff
@@ -7,8 +7,13 @@
 
 import json
 import os
-dir_path = os.path.dirname(os.path.realpath(__file__))
-package_path = os.path.join(dir_path, '..', 'package.json')
-with open(package_path) as f:
-    data = json.load(f)
-OMERO_FIGURE_VERSION = data['version']
+import pkg_resources  # part of setuptools
+
+try:
+    OMERO_FIGURE_VERSION = pkg_resources.require("omero-figure")[0].version
```

Comment:
@sbesson you ok with that? Pretty much now version is only stored in `package.json`